### DISCUSSION TITLE
HTTP file highlight

### DISCUSSION THREAD

**Discussion by LintaoAmons** [mistweaverco/kulala.nvim#63] [General]
How to configure the same highlight colour in the readme

**Comment by gorillamoe** [comment]
`:TSInstall http`

**Reply by LintaoAmons** [reply]
I have ts http installed, but also want to know what theme your are using to have the same color palette

**Reply by gorillamoe** [reply]
Ah, I see. This is what I'm using atm: https://github.com/mistweaverco/bluloco.nvim
